SendWelcomeMessagesActivity/handler.ts

```diff
@@ -93,12 +96,46 @@ Per approfondimenti ti invitiamo a consultare la sezione [servizi](ioit://SERVIC
 `,
         subject: `Quali servizi trovi su IO?`
       }
-      // tslint:disable-next-line:no-identical-functions
-    }).getOrElseL(errs => {
-      throw new Error(
-        "Invalid MessageContent for welcome message: " + readableReport(errs)
-      );
-    })
+    }).getOrElseL(throwInvalidMessageError),
+  CASHBACK: (_: RetrievedProfile) =>
+    NewMessage.decode({
```

Comment:
@Undermaken @CrisTofani can you please check if this is the expected msg format so we can merge it?

Reply:
It doesn't work.
I guess the _frontmatter_ should be placed at the beginning of the message instead of at the end

fyi interest you can test how the app reads and displays the message content
`profile`->`5 taps on app version`->`enable debug`->`markdown playground`->`past your markdown` and check the result

<img src="https://user-images.githubusercontent.com/822471/101043704-e2abe680-357e-11eb-8533-c2bd204be03e.png" height="600" />

Reply:
@Undermaken thanks for the tips, fixed in bfd0a31

Reply:
If this is checked against the in-app markdown editor, we should merge it before the next deploy cc @BurnedMarshal @balanza 

Moreover:

1. check with @Loreau if we can stop sending email for all the welcome messages; note that this will stop **ALL the emails** sent by the service 'IO novità e aggiornamenti'

1. sync with @uolter to add the service ID into the variable `EMAIL_NOTIFICATION_SERVICE_BLACKLIST` in io-fn3-services

1. check with @thisisjp WHEN we must make this message active setting `IS_CASHBACK_ENABLED="true"` into io-fn3-app configuration